ush/setup.py

```diff
@@ -60,7 +62,8 @@ def load_config_for_setup(ushdir, default_config, user_config):
 
     # Load the default config.
     logging.debug(f"Loading config defaults file {default_config}")
-    cfg_d = load_config_file(default_config)
+    #cfg_d = load_config_file(default_config)
```

Comment:
```suggestion
```
Any reason to keep the commented-out code?

Reply:
None!